Source/Mockolate/Setup/ReturnMethodSetup.cs

```diff
@@ -477,6 +539,28 @@ public IReturnMethodSetupCallbackBuilder<TReturn, T1, T2> Do(Action<int, T1, T2>
 		return this;
 	}
 
+	/// <summary>
+	///     Registers a <paramref name="callback" /> to execute when the method is called.
+	/// </summary>
+	public IReturnMethodSetupCallbackBuilder<TReturn, T1, T2> Do<TMock>(Action<TMock, T1, T2> callback)
```

Comment:
Also add an overload that takes an `Action<TMock>` without any additional parameters for all `ReturnMethodSetups`.